functorch/test/test_pythonkey.py

```diff
@@ -590,9 +590,6 @@ def f(x):
         fw_graph, bw_graph = get_fw_bw_graph(f, [torch.randn(5, 5, requires_grad=True)])
         self.assertEqual(get_num_ins_outs(fw_graph), (1, 3))
 
-        ins, outs = get_ins_outs(fw_graph)
-        self.assertEqual(outs[1].target, torch.ops.aten.mm.default)
```

Comment:
With the additional alias node, min_cut_prartitioner is giving a different but still valid result

Reply:
Ugh, I think we need to eventually modify these tests to check memory usage instead of exact ops. Oh well.

Reply:
But, can we fix this check instead of deleting it? I think you might also need to add it here: https://github.com/pytorch/pytorch/blob/master/functorch/_src/partitioners.py#L278

Or maybe under the inductor section.